app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,33 @@
+class CommentsController < ApplicationController
+  # def create
+  #   user = User.find(params[:user_id])
+  #   post = Post.find(params[:post_id])
+  #   @comment = post.comments.new(author_id: current_user, **comment_param)
+  #   if @comment.save
+  #     redirect_to user_post_path(current_user, post)
+  #   else
+  #     redirect_to user_post_path(current_user), alert: 'Failed to add comment!'
+  #   end
+  # end
```

Comment:
- [ ] Kindly avoid leaving commented pieces of code in your codebase. It makes your codebase unclean and is not a good practice :+1: